packages/vscode-extension/src/common/Project.ts

```diff
@@ -38,6 +38,7 @@ export type ProfilingState = "stopped" | "profiling" | "saving";
 export type NavigationHistoryItem = {
   displayName: string;
   id: string;
+  removable?: boolean;
```

Comment:
Does it need to be here? It looks to me like it's only used on frontend side, and you could probably just pass it as a separate prop to `UrlSelectItem`